main.c

```diff
@@ -717,6 +722,26 @@ uint32_t parse_color(const char *color) {
 	return res;
 }
 
+enum slurp_label_anchor parse_label_anchor(const char *anchor) {
```

Comment:
"center-top" is equivalent to "top-center" is equivalent to "center" is equivalent to "top-bottom-left-right-center". 

It will also happily accept something like "top-nonsenes" and treat it the same as "top" or "not-top" is also the same as "top", or "neotop". 

Should this be more strict in what it accepts?

Reply:
Also "top-bottom" is the same as "center" (and similarly "left-right")

Reply:
I thought this lack of strictness simplifies parsing without any harm/conflict. 
The structure of the anchor is the same as used for `wlr_layer_surface` and I just added center as convenience. As such the same duplications happen, but I don't se an issue there.